home/src/main/resources/rdf/tbox/filegraph/vitro-dynamic-api.owl

```diff
@@ -283,6 +284,66 @@
         <rdfs:label xml:lang="en-US">solr query text</rdfs:label>
     </owl:DatatypeProperty>
     
+    <!-- https://vivoweb.org/ontology/vitro-dynamic-api#solrField -->
+    
+    <owl:DatatypeProperty rdf:about="https://vivoweb.org/ontology/vitro-dynamic-api#solrField">
+        <rdfs:subPropertyOf rdf:resource="http://www.w3.org/2002/07/owl#topDataProperty"/>
+        <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#FunctionalProperty"/>
```

Comment:
I think this can't be FunctionalProperty (https://www.w3.org/TR/owl-ref/#FunctionalProperty-def), if it is defined as functionalProperty there can be maximally one value. Similar is for solrFilter, solrFacets, and solrSort.  We might have a problem in validation that rdf/n3 representations of actions are not in accordance with this ontology (if there is more than one solrField data property.